pkg/bootstrap/config.go

```diff
@@ -508,6 +508,9 @@ func getNodeMetaData(envs []string, plat platform.Environment, nodeIPs []string,
 
 	meta.ProxyConfig = (*model.NodeMetaProxyConfig)(pc)
 
+	// Add all instance labels with lower precedence than pod labels
```

Comment:
Could you explain a bit more? I meant instance labels have lower precedence than pod labels so they would be overwritten (upon a key conflict with a pod label) in L522. I'm assuming that there are no conflicts prior to L512, which seems to be true since it would be meta.Labels would be nil

Reply:
Oh I misread it. so the order is ISTIO_META_LABELS < metadata < pod labels - is that correct? We may actually want `metadata < ISTIO_META_LABELS < pod labels` but not a big deal

Reply:
Metadata falls under `meta.PlatformMetadata` while metadata labels and pod labels fall under `meta.Labels`, so they should be separate (unless some later function looks for a key within both, which doesn't happen to my knowledge). Within `meta.Labels` we have metadata labels < pod labels.